src/viur/core/bones/base.py

```diff
@@ -1200,8 +1199,15 @@ def _compute(self, skel: 'viur.core.skeleton.SkeletonInstance', bone_name: str):
         compute_fn_parameters = inspect.signature(self.compute.fn).parameters
         compute_fn_args = {}
         if "skel" in compute_fn_parameters:
-            cloned_skel = skel.clone()
-            cloned_skel[bone_name] = None  # remove value form accessedValues to avoid endless recursion
+
+            if skel.skel_type == "RefSkel":  # we have a ref skel we must load the complete skeleton
```

Comment:
I think the attribute `skel_type` is not necessary:

```suggestion
            if issubclass(skel.skeletonCls, RefSkel):  # we have a ref skel we must load the complete skeleton
```